templates/blocks/review_item.liquid

```diff
@@ -0,0 +1,23 @@
+<div class="reviews-wrapper">
+  <div class="ratings-wrapper sc-flex-col sc-justify-center sc-align-items-center">
+    <div class="reviews-rating">
+      {%- render 'shared/icons/star' %}
+      {%- render 'shared/icons/star' %}
+      {%- render 'shared/icons/star' %}
+      {%- render 'shared/icons/star' %}
+      {%- render 'shared/icons/star' %}
+    </div>
+
+    {%- if content_block.title != blank %}
+      <p class="reviews-message sc-m-none sc-font-medium">
+        {{ content_block.title }}
+      </p>
+    {%- endif %}
+
+    {%- if content_block.subtitle != blank %}
+      <p class="sc-m-none sc-font-medium sc-font-bold">
```

Comment:
Add a child component class here too, even if we are not going to use it. It will help us to understand the component's structure overall.